docs/contributing/developing.md

```diff
@@ -1,4 +1,4 @@
-(developing)=
+(developing-wagtail)=
```

Comment:
Had to change this reference since I kept getting this error:

`/code/wagtail/docs/contributing/index.md:17: WARNING: more than one target found for 'myst' cross-reference developing: could be :std:ref:`developing docs` or :std:doc:`developing docs` [myst.ref]`

Reply:
The other way to fix this error is to reference the `developing.md` in the table of contents on index. 

Note the file extension usage.